torch/distributed/nn/__init__.py

```diff
@@ -1,4 +1,14 @@
 import torch
 if torch.distributed.rpc.is_available():
     from .api.remote_module import RemoteModule
-from .functional import *  # noqa: F403
+from .functional import (
```

Comment:
The problem is that changes like this in public APIs are technically BC-breaking.
It's fine to do in testing files and testing support. But anything else is more dangerous. You will need a very thorough review from the owner of each of these files.
If you want to merge this quickly, you might want to remove these from the PR and only fix this in testing for now.